src/twisted/defer/newsfragments/10222.bugfix

```diff
@@ -0,0 +1 @@
+always consume the result of awaited Deferreds
```

Comment:
This reads more like a commit message than a news fragment. Who are you commanding to always consume the result of an awaited deferred? the user? what consumes it? how?

Reply:
(i.e. when someone reads this in a changelog, how do they understand what behavior has actually changed?)

Reply:
```suggestion
``Deferred.fromCoroutine`` (and therefore also ``ensureDeferred``) now always consumes the result of awaited Deferreds. Previously awaiting the same Deferred synchronously multiple times in a row would resolve to the same result.
```